io.catenax.pcf_for_battery_component_supplier/1.0.0/ProductCarbonFootprintForBatteryComponentSupplier.ttl

```diff
@@ -0,0 +1,220 @@
+
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.pcf_for_battery_component_supplier:1.0.0#> .
+@prefix ext-bpn: <urn:samm:io.catenax.shared.business_partner_number:2.0.0#> .
+@prefix ext-pcf: <urn:samm:io.catenax.pcf:9.0.0#> .
+@prefix ext-serial-part: <urn:samm:io.catenax.serial_part:3.0.0#> .
+@prefix samm-u: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+
+:ProductCarbonFootprintForBatteryComponentSupplier a samm:Aspect ;
+   samm:properties ( :cxPCFValues ext-pcf:secondaryEmissionFactorSources [ samm:property ext-pcf:primaryDataShare; samm:optional true ] :customerPartId :technologicalRepresentativeness :geographicalRepresentativeness :timerelatedRepresentativeness :dataQualityRating :manufacturerPartId :validityPeriod :siteBpn :legalBpn :productQualityParameter :contentC02Metals ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:cxPCFValues a samm:Property ;
```

Comment:
bad practice: do not use "Catena-X" in descriptions, "cx" element names etc. : this means that it is not reusable in any other battery passport in any other dataspace or company